docs/source/example_notebooks/dowhy_simple_example.ipynb

```diff
@@ -367,15 +367,18 @@
    "source": [
     "res_subset=model.refute_estimate(identified_estimand, estimate,\n",
     "        method_name=\"data_subset_refuter\", subset_fraction=0.9)\n",
-    "print(res_subset)\n"
+    "print(res_subset)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "As you can see, the propensity score stratification estimator is reasonably robust to refutations.\n",
-    "For reproducibility, you can add a parameter \"random_seed\" to any refutation method, as shown below."
+    "\n",
+    "**Reproducability and Parallelization**: For reproducibility, you can add a parameter \"random_seed\" to any refutation method, as shown below.\n",
+    "\n",
+    "You can also use built-in parallelization to speed up the refutation process. Simply set `n_jobs` to a value greater than 1 to spread the workload to multiple CPUs, or set `n_jobs=-1` to use all CPUs. "
```

Comment:
Added some text here about parallelization. It's only a really small hint though so maybe a bit too easy to overlook? I do think the feature is quite useful since it can save a lot of time so it should be used a lot too, at least I hope.

Reply:
that looks good!